### DISCUSSION TITLE
Week 5 - Project Extended Thoughts and Ideas

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#117] [General]
## A Time and Project Management Website
 - One of the thoughts I had about this time management system is to go past the basic task management system and instead provide instructional training to the user as they use the tool to track their daily tasks and project deliverables. Things like prompting each Monday morning to set weekly goals and then encouraging them to add tasks to their list to help move toward completing the goals. Helping the user prioritize their day by asking them to rank the importance and relevance of these tasks using a system similar to Stephen Covey's prioritization matrix. By providing teaching moments while accomplishing their daily tasks, I could help to motivate them to use better prioritization mechanisms.

 - The thought is to use HTML and CSS to build a series of pages that will display a user's daily activities. The pages will include a daily activity and time tracker, a project summary, and a status report page, along with a series of instructional help modules that will help with goal setting and progress trackers.

 - Javascript is going to be needed to capture new task activities and add them to a storage array using simple local storage for now (maybe adding a SQL db sometime in the future). The activity tracker will also need a lot of javascript to calculate task time and then total activity time.

## Microsite for a new product pre-launch (fake product like GPS Running Shoes or something silly like that)
- This section would be a lot of fun to work on since I am a marketer at heart. With launching a new product, we would want a "wow" landing page that would introduce the new product along with some product specifications and customer tutorials. Like a lot of launches, we would want to include introductory offers and promotional pricing.

- As with some products that need more direct servicing support, I would include a "get more" information form that would mock up a submission to a CRM and marketing automation system.

- If I was to mock up a GPS shoe, I would want to make sure that the order form would allow the customer to select the type of shoe color and size and then help to calculate the order total before submitting the form to the fake order team.

**Comment by metasean** [comment]
I love your breakouts and both projects have a lot of potential!
